lib/iris/tests/test_grib_save.py

```diff
@@ -34,7 +34,8 @@
 
 if tests.GRIB_AVAILABLE:
     import gribapi
-
+    from iris.fileformats.grib._load_convert import _MDIs
+    _mdis = [mdi for mdi in _MDIs]
```

Comment:
Could you not just use `MDIs` directly? This seems like an unnecessary variable definition.